std/conv.d

```diff
@@ -176,8 +176,9 @@ class ConvOverflowException : ConvException
 }
 
 /**
-The `to` template converts a value from one type _to another.
-The source type is deduced and the target type must be specified, for example the
+$(D_CODE T $(DDOC_PSYMBOL _to)$(DDOC_TEMPLATE_PARAM_LIST (S))(S value);)
```

Comment:
![image](https://user-images.githubusercontent.com/4370550/35538786-5b162198-054f-11e8-8735-11618a337f78.png)


There's any precedent for this, is there?